products/nativescript/market_helpers.py

```diff
@@ -56,20 +81,84 @@ def get_data_market():
 
         return testing_data
 
+    @staticmethod
+    def get_preserved_data():
+        file_path = os.path.join(Settings.TEST_RUN_HOME, 'results.json')
+        data = None
+        if os.path.isfile(file_path):
+            with open(file_path, "r") as json_file:
+                data = json.load(json_file)
```

Comment:
raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded